ui/js/perf.js

```diff
@@ -290,7 +290,7 @@ perf.factory('PhCompare', [ '$q', '$http', '$httpParamSerializer', 'thServiceDom
                                         timeRange = Math.round(now - timeRange);
 
                                         //now figure out which predefined set of data we can query from
-                                        var timeRange = _.find(phTimeRanges, function(i) { return timeRange <= i.value; });
+                                        timeRange = _.find(phTimeRanges, function(i) { return timeRange <= i.value; });
```

Comment:
I think I would fix this by changing the variable name from timeRange to phTimeRange, as your change makes us use the same variable to represent two different things.